### DISCUSSION TITLE
Trouble Verifying Academic Affiliation

### DISCUSSION THREAD

**Discussion by amethyst0329** [community/community#58058] [GitHub Education]
### Hi everyone,

I am having trouble verifying my academic affiliation. I have submitted for verification 4 times and I get the same email:

Hey amethyst0329,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Jun 14, 2023 for [@amethyst0329](https://urldefense.com/v3/__https://github.com/amethyst0329__;!!BeImMA!6O0gkKDc2-mtOt5z8NV2MK8ROcFFrnmmGAUKLu46F3UbicYiKP56xHX7-5-S9e8CHCB-pWVB8MMkAGjmDs2ZYJ1tPHs$).

The text in the photo you used was unable to be read. If using your device camera, make sure the image is clear and legible. For some languages, you may need to capture the original document beside an English translation.

You are unlikely to be verified until you have completed your [GitHub user profile](https://urldefense.com/v3/__https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile__;!!BeImMA!6O0gkKDc2-mtOt5z8NV2MK8ROcFFrnmmGAUKLu46F3UbicYiKP56xHX7-5-S9e8CHCB-pWVB8MMkAGjmDs2Z_GHgdDs$) with your full name as it appears in your academic affiliation document plus a short bio.

The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

Your document does not appear to include a date demonstrating current academic affiliation. For countries utilizing non-standard calendars, you may need to capture the original document beside one with a converted date. You may include multiple documents in your image, so long as they are legible.

I have submitted:
1. A previously saved enrollment verification
2. A newly created enrollment verification
3. A transcript
4. An updated profile (picture, bio, legal name) + a different picture of my transcipt.

Each document has the school's name and official logo. I am using my school email on my profile. I don't know what else I should do to fix this.
Any suggestions?

-Amethyst

**Comment by samus-aran** [comment]
Hi @amethyst0329 

I suspect that the image submitted had trouble being picked up by the system - maybe too large? Try submitting a smaller resolution or image filesize. 

Let us know how you get on 👍
